pkg/spec/tf_job.go

```diff
@@ -11,20 +11,18 @@ import (
 )
 
 const (
-	TPRKind        = "tf-job"
-	TPRKindPlural  = "tfjobs"
-	TPRGroup       = "mlkube.io"
-	TPRVersion     = "v1beta1"
-	TPRDescription = "TensorFlow training job"
+	CRDKind        = "tf-job"
```

Comment:
Thanks; you're right we don't use it anymore. The only thing I specify is the PluralKind which I specify as "tfjobs". This ends up producing

spec:
    group: mlkube.io
    names:
      kind: TfJob
      listKind: TfJobList
      plural: tfjobs
      singular: tfjob
    scope: Namespaced
    version: v1beta1
  status:
    acceptedNames:
      kind: TfJob
      listKind: TfJobList
      plural: tfjobs
      singular: tfjob

This looks correct; i.e. the plural name are all lower case no hypens which follows the CronTab example in the docs
https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/